src/main/scala/amailp/intellij/robot/psi/reference/LibraryToDefinitionReference.scala

```diff
@@ -21,25 +22,25 @@ class LibraryToDefinitionReference(element: PsiElement)
     (for {
       lib <- currentRobotFile.getImportedLibraries
     } yield LookupElementBuilder
-      .create(lib.getText)
+      .create(lib.getName)
       .withCaseSensitivity(false)
       .withTypeText("Library", true)
       .withAutoCompletionPolicy(AutoCompletionPolicy.GIVE_CHANCE_TO_OVERWRITE)
       .asInstanceOf[AnyRef])(breakOut)
 
   override def multiResolve(incompleteCode: Boolean): Array[ResolveResult] = {
-    val qNameLast = QualifiedName.fromDottedString(element.getText).getLastComponent
+    val qNameLast = QualifiedName.fromDottedString(element.asInstanceOf[LibraryValue].getName).getLastComponent
```

Comment:
I do not like all these `asInstanceOf`. We could change the type of element to `LibraryValue` and to the `asInstanceOf` only once in `LibraryReferenceContributor`, where `LibraryToDefinitionReference` is created.